src/lib.rs

```diff
@@ -1229,5 +1232,13 @@ impl<K, V> Table<K, V> {
     }
 }
 
+#[inline]
+/// Returns the number of physical CPUs in the machine (_O(1)_).
+fn get_ncpu() -> usize {
```

Comment:
Why is all the `unsafe` needed here?

Reply:
Because to avoid calling `num_cpus::get_physical` multiple times I'm storing it's return value in a `static mut`, which can only be read or mutated from unsafe code (I think).

Reply:
I'm pretty sure it's safe to read without `unsafe`. Modifying it you might be right about though. Maybe make it an `AtomicUsize`, which would get rid of all the `unsafe`?

Reply:
Dunno, `AtomicUsize` doesn't make much sense to me, since it implies the value will be persistently modified (for me atleast). Still, I could use it if that's your decision.

Reply:
I would prefer it because it removes some unnecessary `unsafe`. One day, we'll get something like [this](https://github.com/rust-lang/rfcs/pull/2788) in the standard library, and then we could move to that instead :)

Reply:
Wow, that type is preciselly what we need! Alright then, I'll go with `AtomicUsize` then.

Reply:
What ordering should I go for thought?

Reply:
`Relaxed` should be fine in this instance. Even _if_ a read ends up returning the original `0` value, you'll still end up with reasonable behavior (using `MIN_TRANSFER_STRIDE`).